pam/vplugin/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package vplugin contains the details for communicating with a plugin-based PAM implementation.
```

Comment:
`vplugin.go` has some package-level comments. Perhaps remove that when this is added.

Reply:
Ah my bad, I somehow missed it